### PR TITLE
Tabs pattern updates to new IA

### PR DESCRIPTION
**Pull request by chri2547**
@araiford, tabs is ready for final review and merge

image

### CODE REVIEW

File: _component_design/tabset.md [hunk 1]
```diff
@@ -105,22 +170,38 @@ Pad the top and bottom of the content area by 24px. Pad the sides of the content
...
+<section class="static-section" markdown="1">
+
 <div class="hxRow"  markdown="1">
 
 {% column left:"hxCol hxSpan-12-xs hxSpan-12-sm hxSpan-4-md hxSpan-4-lg" %}
 
 ### Hover state
 
-Hovering over another tab changes the color of the text. Hovering off the tab changes it back to the default color.
+Construct the tab so that the text color changes to hex color #16b9d4 when a user hovers over the tab.
```
Comment (palindromebart) [inline _component_design/tabset.md]: When the cursor moves over any part of an unselected tab, change to the hover state. when the cursor moves off of the tab, the text should change back to the default color.

File: _component_design/tabset.md [hunk 2]
```diff
@@ -105,22 +170,38 @@ Pad the top and bottom of the content area by 24px. Pad the sides of the content
...
+
+When a user moves off the tab, the text should change back to the default color.
```
Comment (palindromebart) [inline _component_design/tabset.md]: remove this sentence

File: _component_design/tabset.md [hunk 3]
```diff
@@ -137,14 +218,15 @@ Hovering over another tab changes the color of the text. Hovering off the tab ch
 
 ### Pressed state
 
-Clicking the text on the tab changes the color by a few shades to visually confirm an action.
+To visually confirm to a user that a tab is selected, change the color of the tab text by a few shades.
```
Comment (palindromebart) [inline _component_design/tabset.md]: To visually reinforce the tab selection, change the color of the tab text to the pressed state for the duration of the click. Change the tab text to selected state after the click.

**Comment by cyrichardson** [comment]
@chri2547 Suggestions shortly...

**Comment by palindromebart** [comment]
LGTM

File: _component_design/tabset.md [hunk 4]
```diff
@@ -17,34 +17,95 @@ helix-ui-javascript: true
...
+- The default view must display the left-most tab.
+- Present tabs as a single row.
+- Do not construct nested tabs that include another set of tabbed content within a tab.
+- Highlight the tab corresponding to the visible content.
+- Provide all relevant contextual information and interface items so a user can perform a single task within a tab.
+- Do not use tabs for primary navigation.
+- Clearly name tabs to differentiate them. Names should be short, scannable, and preferably one word.
+- Capitalize the first letter of each word in a tab name.
+- If necessary, wrap tab names to a second line. Truncate tab names that exceed two lines.
+
```
Comment (araiford) [inline _component_design/tabset.md]: add:

`Don't use tab sets in multiple areas of the same page.`

File: _component_design/tabset.md [hunk 5]
```diff
@@ -17,34 +17,95 @@ helix-ui-javascript: true
...
+Consider the following best practices when you construct tabs:
+
+- Only one active tab per page is allowed.
+- The default view must display the left-most tab.
+- Present tabs as a single row.
+- Do not construct nested tabs that include another set of tabbed content within a tab.
+- Highlight the tab corresponding to the visible content.
+- Provide all relevant contextual information and interface items so a user can perform a single task within a tab.
+- Do not use tabs for primary navigation.
```
Comment (araiford) [inline _component_design/tabset.md]: This line can be removed since it repeats line 34.

File: _component_design/tabset.md [hunk 6]
```diff
@@ -17,34 +17,95 @@ helix-ui-javascript: true
...
+
+Use tabs in the following situations:
+
+- Users need to alternate between different views, while the context remains constant.
+- You want the content of a page to be scannable and easy to navigate. Tabs help users quickly find what they need.
+- You do not need to create a single, unified experience for the user. Tabs eliminate the need for the user to scroll through long pages to find the right section.
+
+Do not use tabs for the following situations:
+
+- Tabs are not suited for use for navigation
```
Comment (araiford) [inline _component_design/tabset.md]: Consider changing to something like: 

`Tabs are not suited for use as primary navigation for an entire page.`

File: _component_design/tabset.md [hunk 7]
```diff
@@ -17,34 +17,95 @@ helix-ui-javascript: true
...
+
+- Only one active tab per page is allowed.
+- The default view must display the left-most tab.
+- Present tabs as a single row.
+- Do not construct nested tabs that include another set of tabbed content within a tab.
+- Highlight the tab corresponding to the visible content.
+- Provide all relevant contextual information and interface items so a user can perform a single task within a tab.
+- Do not use tabs for primary navigation.
+- Clearly name tabs to differentiate them. Names should be short, scannable, and preferably one word.
+- Capitalize the first letter of each word in a tab name.
```
Comment (araiford) [inline _component_design/tabset.md]: This should probably say:
`Tab names must follow title case capitalization.`

File: _component_design/tabset.md [hunk 8]
```diff
@@ -17,34 +17,95 @@ helix-ui-javascript: true
...
+- If necessary, wrap tab names to a second line. Truncate tab names that exceed two lines.
```
Comment (araiford) [inline _component_design/tabset.md]: I strongly disagree with this line.  Suggest:

`Avoid using lengthy tab names.  Do not allow text to wrap or truncate.`

**Pull request merged**